### PR TITLE
Adapt tooltip for target label

### PR DESCRIPTION
**Pull request by reichlfl**
The target label now shows details about instance and process during an
active connection. This is mainly used in E2E tests so we can verify
the instance ID, but will also help users to make sure they are
profiling the correct process.

Bug: b/232251224
Test: Adapted E2E test (will add in a follow-up PR)

### CODE REVIEW

File: src/SessionSetup/TargetLabel.cpp [hunk 1]
```diff
@@ -117,18 +117,19 @@ void TargetLabel::ChangeToStadiaTarget(const StadiaTarget& stadia_target) {
...
+  machine_ = instance.display_name;
+  SetProcessCpuUsageInPercent(process.cpu_usage());
   ui_->targetLabel->setVisible(true);
   ui_->fileLabel->setVisible(false);
+
+  ui_->targetLabel->setToolTip(
+      QString{"Connection active.<br/><br/>"
+              "Instance: %1 (%2)<br/>"
+              "Process: %3 (%4)"}
+          .arg(instance.display_name, instance.id, QString::fromStdString(process.name()),
```
Comment (vwbaker) [inline src/SessionSetup/TargetLabel.cpp:131]: nit; QString::fromStdString(process.name()) -> process_

**Review by vwbaker — APPROVED** [review]

**Review by antonrohr — APPROVED** [review]
LGTM, thanks for the extensive unit tests

File: src/SessionSetup/TargetLabelTest.cpp [hunk 2]
```diff
@@ -36,17 +38,44 @@ TEST(TargetLabel, ChangeToFileTarget) {
   EXPECT_EQ(label.GetFilePath().value(), path);
 }
 
+const char* kProcessName = "test process";
+const char* kInstanceName = "test instance";
+const double kCpuUsage = 50.1;
+const char* kCpuUsageDisplay = "50";
+
+void ChangeToFakeStadiaTarget(TargetLabel& label) {
```
Comment (antonrohr) [inline src/SessionSetup/TargetLabelTest.cpp:46]: static

File: src/SessionSetup/TargetLabelTest.cpp [hunk 3]
```diff
@@ -36,17 +38,44 @@ TEST(TargetLabel, ChangeToFileTarget) {
...
+                                                  "ipAddress: \"127.0.0.1\", "
+                                                  "state: \"IN_USE\", "
+                                                  "owner: \"unit@test\", "
+                                                  "lastUpdated: \"2022-07-04T13:22:04Z\", "
+                                                  "pool: \"unit-test-pool\""
+                                                  "}"}
+                                              .arg(kInstanceName)
+                                              .toUtf8());
+
+  assert(maybe_instance.has_value());
```
Comment (antonrohr) [inline src/SessionSetup/TargetLabelTest.cpp:67]: this should be an ASSERT_TRUE

Comment (antonrohr) [reply src/SessionSetup/TargetLabelTest.cpp:67]: Or even better:
`EXPECT_THAT(maybe_instance, orbit_test_utils::HasValue());`
Then you get a better error message in the tests

**Review by antonrohr — APPROVED** [review]
LGTM (assuming the tests are green)

**Pull request merged**